configuration/config.yml

```diff
@@ -15,5 +15,8 @@ analysers_settings:
     - requirements: "stats"
       scripts: 
```

Comment:
I would have the name of the metrics, identifying the document. Something like:

```yaml
scripts:
   ram: 
      script_trial: "/app/analysers/trials/ram.py"
      script_experiment: "/app/analysers/experiments/ram.py"
      packages: "TargetHolding:pyspark-cassandra:0.2.2"
```

and so on for the other metrics